apache-kafka/start_service.sh

```diff
@@ -0,0 +1,5 @@
+zookeeper-server-start /usr/local/etc/kafka/zookeeper.properties
```

Comment:
For a script, you should add the `#!/usr/bin/env bash` on the first line and make the file executable